src/main/java/uk/gov/hmcts/darts/common/entity/TranscriptionWorkflowEntity.java

```diff
@@ -0,0 +1,48 @@
+package uk.gov.hmcts.darts.common.entity;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.JoinColumn;
+import jakarta.persistence.ManyToOne;
+import jakarta.persistence.SequenceGenerator;
+import jakarta.persistence.Table;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.Setter;
+import uk.gov.hmcts.darts.common.entity.base.CreatedModifiedBaseEntity;
+
+@Entity
+@Table(name = TranscriptionWorkflowEntity.TABLE_NAME)
+@Getter
+@Setter
+@EqualsAndHashCode(callSuper = false)
+public class TranscriptionWorkflowEntity extends CreatedModifiedBaseEntity {
+
+    public static final String TABLE_NAME = "transcription_workflow";
+    public static final String TRANSACTION_WORKPLACE_ID = "trw_id";
+    public static final String TRANSACTION_WORKPLACE_GENERATOR = "trw_gen";
+    public static final String TRANSACTION_WORKPLACE_SEQUENCE = "trw_seq";
+    public static final String TRANSACTION_ID = "tra_id";
+    public static final String WORKFLOW_STAGE = "workflow_stage";
+    public static final String WORKFLOW_COMMENT = "workflow_comment";
+
+    @Id
+    @Column(name = TRANSACTION_WORKPLACE_ID)
+    @GeneratedValue(strategy = GenerationType.SEQUENCE, generator = TRANSACTION_WORKPLACE_GENERATOR)
+    @SequenceGenerator(name = TRANSACTION_WORKPLACE_GENERATOR, sequenceName = TRANSACTION_WORKPLACE_SEQUENCE, allocationSize = 1)
+    private Integer id;
+
+    @ManyToOne(optional = false)
+    @JoinColumn(name = TRANSACTION_ID, nullable = false)
+    private TranscriptionEntity transcription;
+
+    @Column(name = WORKFLOW_STAGE, nullable = false)
+    private String workflowStage;
```

Comment:
I wish we had a better understanding of what should be here - whether an enum and how that might be different from the Transcription Status which the 5 I chose were based originally on the heritage workflow lifecycle events anyway e.g. Requested, Awaiting Authorisation, With Transcriber, Complete, Rejected.

Wondering, do we want an enum here with the names you have similar to `TranscriptionStatusTypeEnum`?
@jackmaloney / @hemantasharma1129

Reply:
Yes, that sounds like a plan @richardbull. Mind creating a ticket and making sure they align with what Barry is putting in? We also need 'Closed' as a stage for modernised.

Reply:
@karen-hedges Please can you add an enum here to add the similar String enumerated type for this column?

    REQUESTED,
    AWAITING_AUTHORISATION,
    WITH_TRANSCRIBER,
    COMPLETE,
    REJECTED,
    CLOSED;


It would be similar to what we did in `MediaRequestEntity`
```
    @Column(name = ..., nullable = false)
    @Enumerated(EnumType.STRING)
    private WorkflowStageEnum workflowStage;
```